src/community/advanced-data-card.tsx

```diff
@@ -0,0 +1,462 @@
+            >
+                {hasButton && (
+                    <div
+                        tabIndex={-1}
+                        className={classNames(
+                            sprinkles({
+                                display: 'flex',
+                            }),
+                            styles.marginTopButton
+                        )}
+                        style={{
+                            marginRight: 16,
+                            position: 'relative',
+                        }}
+                    >
+                        {button}
+                    </div>
+                )}
+                <div
+                    className={hasAllItens ? styles.marginRightAuto : ''}
+                    style={{
+                        display: 'flex',
+                        flexDirection: 'row',
+                        marginTop: hasAllItens ? 8 : 16,
+                        flexWrap: 'wrap',
+                    }}
+                >
+                    {hasFooterImage && (
+                        <Box paddingRight={16} className={sprinkles({alignItems: 'center', display: 'flex'})}>
+                            {footerImage}
+                        </Box>
+                    )}
+
+                    {hasFooterText && (
+                        <div style={{maxWidth}} className={styles.footerText}>
+                            <Text2 truncate={footerTextLinesMax} regular>
+                                {footerText}
+                            </Text2>
+                        </div>
+                    )}
+                </div>
+                {hasButtonLink && (
+                    <div
+                        tabIndex={-1}
+                        className={classNames(
+                            sprinkles({
+                                display: 'flex',
+                            }),
+                            hasAllItens ? styles.marginTop : styles.marginTopButton
+                        )}
+                        style={{
+                            position: 'relative',
+                            marginLeft: -12,
+                            marginRight: -12,
+                        }}
+                    >
+                        {buttonLink}
+                    </div>
+                )}
+            </div>
+        </>
+    );
+};
+
+type ExtraTypeof = typeof StackingGroup;
+
+type TextAs = 'h1' | 'h2' | 'h3' | 'h4';
+
+interface AdvancedDataCardProps {
+    onPress?: () => void;
+    stackingGroup?: RendersNullableElement<typeof StackingGroup>;
+    headline?: RendersNullableElement<typeof Tag>;
+    pretitle?: string;
+    pretitleLinesMax?: number;
+    pretitleAs?: TextAs;
+    title?: string;
+    titleLinesMax?: number;
+    titleAs?: TextAs;
+    subtitle?: string;
+    subtitleLinesMax?: number;
+    description?: string;
+    descriptionLinesMax?: number;
+    extra?: Array<RendersNullableElement<ExtraTypeof>>;
+    smallSlotSpace?: boolean;
```

Comment:
I'd rename this:
```suggestion
    extraDividerPadding?: 8 | 24;
```
With 24 as default